lib/kafo_parsers/version.rb

```diff
@@ -1,3 +1,3 @@
 module KafoParsers
-  VERSION = "1.1.2"
+  VERSION = "1.1.3"
```

Comment:
You missed the actual code change :)

There is actually a config file:
https://github.com/theforeman/kafo_parsers/blob/master/.bumpversion.cfg

So you can run `bumpversion minor` and it'll do the right thing for a minor release. Including tagging if you want.

Reply:
I missed committing it. That code change was sitting there waiting for me to come back and find it.